cincinnati/src/plugins/internal/arch_filter.rs

```diff
@@ -87,6 +91,10 @@ lazy_static! {
 #[async_trait]
 impl InternalPlugin for ArchFilterPlugin {
     async fn run_internal(self: &Self, internal_io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("arch-filter".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```

Reply:
I'll keep this one not merged yet, as it probably would be replaced by `interface.rs`/`plugins/mod.rs` modifications